app/src/main/java/org/gem/indo/dooit/views/settings/SettingsActivity.java

```diff
@@ -26,14 +30,16 @@
 /**
  * Created by Bernhard Müller on 2016/07/22.
  */
-public class SettingsActivity extends AppCompatActivity {
+public class SettingsActivity extends DooitActivity {
```

Comment:
`fontPath` was not working because the settings activity did not inherit from DooitActivity, so the Calligraphy context wrapper wasn't being set up.